server/constants/routes.go

```diff
@@ -2,18 +2,21 @@ package constants
 
 const (
 	// Plugin API Routes
-	APIPrefix                = "/api/v1"
-	WildRoute                = "{anything:.*}"
-	PathOAuthConnect         = "/oauth/connect"
-	PathOAuthCallback        = "/oauth/complete"
-	PathGetAllLinkedProjects = "/project/link"
-	PathUnlinkProject        = "/project/unlink"
-	PathUser                 = "/user"
-	PathCreateTasks          = "/tasks"
-	PathLinkProject          = "/link"
+	APIPrefix                    = "/api/v1"
+	WildRoute                    = "{anything:.*}"
+	PathOAuthConnect             = "/oauth/connect"
+	PathOAuthCallback            = "/oauth/complete"
+	PathGetAllLinkedProjects     = "/project/link"
+	PathUnlinkProject            = "/project/unlink"
+	PathUser                     = "/user"
+	PathCreateTasks              = "/tasks"
+	PathLinkProject              = "/link"
+	PathCreateSubscriptions      = "/subscriptions"
```

Comment:
No need to have 2 constant with same values 
```suggestion
	PathSubscriptions      = "/subscriptions"
```